packages/eas-cli/src/user/SessionManager.ts

```diff
@@ -144,13 +145,15 @@ export default class SessionManager {
    *
    * @deprecated Should not be used outside of context functions, except in the AccountLogin command.
    */
-  public async showLoginPromptAsync({
-    nonInteractive = false,
-    printNewLine = false,
-  } = {}): Promise<void> {
+  public async showLoginPromptAsync(
+    { nonInteractive = false, printNewLine = false } = {},
+    sso?: boolean | undefined
```

Comment:
Add to existing named options object rather than a new param

Reply:
Done